tools/downloader/list_topologies.yml

```diff
@@ -3233,3 +3233,125 @@ topologies:
     output: "Security/object_detection/barrier/0106/dldt/"
     framework: dldt
     license: https://raw.githubusercontent.com/opencv/open_model_zoo/master/LICENSE
+
+  - name: "mobilenet-v2-pytorch"
+    description: >-
+      MobileNet V2 is image classification model pretrained on ImageNet dataset. This
+      is a PyTorch implementation of MobileNetV2 architecture as described in
+      the paper "Inverted Residuals and Linear Bottlenecks: Mobile Networks for Classification,
+      Detection and Segmentation" <https://arxiv.org/pdf/1801.04381.pdf>.
+
+      The model input is a blob that consists of a single image of "1x3x224x224"
+      in RGB order. 
+
+      The model output is typical object classifier for the 1000 different classifications
+      matching with those in the ImageNet database.
+    task_type: "classification"
+    files:
+      - name: MobileNetV2.py
+        sha256: 7071e01e07e5f1b6127a2628d1be9c65cc20edfa745f4f4eb6354e7b27f5b444
+        source: https://raw.githubusercontent.com/tonylins/pytorch-mobilenet-v2/fa86a85e4026caa217fe8d62f370848195cedf4b/MobileNetV2.py
+      - name: mobilenet-v2.pth.tar
```

Comment:
This doesn't seem to actually be a tar file, so to avoid confusion, I think you should remove the `.tar` extension.